FIX: Rename file

talzid opened the pull request.

Commit:
FIX: Rename file

tancnle commented:
@talzid Sorry I missed the original PR. Since this falls under training, could we nest under the `training` folder?

```
❯ tree
.
├── README.md
└── docs
    ├── training
    │   └── markdown.md
```

Commit:
FIX: moving markdown-guide to training folder

Markdown guide is a training document, and therefore should be in training, not documentation

tancnle reviewed (approved):
Thanks, @talzid. Looks great to me 🚀

ghost commented:
closing this request as it might affect @kats8 's content changes.
